kuma/users/views.py

```diff
@@ -467,14 +470,53 @@ def user_edit(request, username):
 
             return redirect(edit_user)
 
+    subscription_info = None
+    stripe_customer = get_stripe_customer(edit_user)
```

Comment:
What I meant to ask with my comment before, can we extract that whole block 474-510 into its own method, similar to before? i.e. something like `retrieve_and_synchronize_subscriptions`?
Personally I like shorter methods which extract out what can be meaningfully named and extracted, just want to get a feel for if you feel similar or whether it's my personal preference.

Reply:
Yep yep. I gotcha. 
What I didn't like before was that view related stuff was put into the `stripe_utils.py` when it wasn't a utility, but actually totally related to the view part. Now, the edit user view function has become huge which isn't great. I'll refactor it a bit.

Reply:
Addressed in the latest commit.